apps/nextcloud/30.0.3/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nextcloud:
-    image: linuxserver/nextcloud:30.0.3
+    image: linuxserver/nextcloud:30.0.4
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
There is a small difference in the versions of the `nextcloud` Docker image being used from `30.0.3` to `30.0.4`. This should not have significant implications unless you have specific requirements or configurations that depend on those differences. If no changes are needed, this change can be considered an update for better security patches and feature improvements. For future deployment, ensure consistency between containers if updates are necessary.